v20.2/export.md

```diff
@@ -30,14 +30,18 @@ Only members of the `admin` role can run `EXPORT`. By default, the `root` user b
 
  Parameter | Description
 -----------|-------------
- `file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported CSV data.<br><br>Note: Exports do not generate unique names across exports, so each export should have a unique destination to avoid overwriting.
+ `file_location` | Specify the [URL of the file location](#export-file-url) where you want to store the exported CSV data.<br><br>Note: It is best practice to use a unique destination for each export, to avoid the potential for multiple exports in one directory.
  `WITH kv_option` | Control your export's behavior with [these options](#export-options).
  `select_stmt` | Specify the query whose result you want to export to CSV format.
  `table_name` | Specify the name of the table you want to export to CSV format.
 
 ### Export file URL
 
-You can specify the base directory where you want to store the exported .csv files. CockroachDB will create the export file(s) in the specified directory with programmatically generated names (e.g., n1.1.csv, n1.2.csv, n2.1.csv, ...). Each export should have a unique destination to avoid overwriting other exports.
+You can specify the base directory where you want to store the exported .csv files. CockroachDB will create the export file(s) in the specified directory with programmatically generated names (e.g., `exportabc123-n1.1.csv`, `exportabc123-n1.2.csv`, `exportabc123-n2.1.csv`, ...). Each export should use a unique destination directory to avoid collision with other exports.
+
+{{site.data.alerts.callout_info}}
+<span class="version-tag">New in v20.2:</span> A hexadecimal hash code (`abc123...` in the file names above) uniquely identifies each export _run_; files sharing the same hash were part of the same export. If you see multiple hash codes within a single destination directory, then the directory contains multiple exports, which will likely cause confusion (duplication) on import. You may wish to manually clean up the directory, to ensure that it contains only a single export run.
```

Comment:
line spacing on the preview looks a little off, but otherwise LGTM!

Reply:
Agree, will make that a separate PR, or perhaps just an issue -- I probably don’t want to mess with docs CSS.